### PR TITLE
Freemium

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- add freemium ("free") plan
- the schema and validator depends on the path (which is connected to the provider)

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/cmd/broker/main.go [hunk 1]
```diff
@@ -288,13 +293,13 @@ func main() {
 	defaultPlansConfig, err := servicesConfig.DefaultPlansConfig()
 	fatalOnError(err)
 
-	plansValidator, err := broker.NewPlansSchemaValidator(defaultPlansConfig)
-	fatalOnError(err)
+	//plansValidator, err := broker.NewPlansSchemaValidator(defaultPlansConfig)
```
[the remark this answers is not in the document]
Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/cmd/broker/main.go]: fixed

File: components/kyma-environment-broker/cmd/broker/main.go [hunk 2]
```diff
@@ -139,8 +140,11 @@ type Config struct {
 	TrialRegionMappingFilePath string
 	MaxPaginationPage          int `envconfig:"default=100"`
 
-	LogLevel   string `envconfig:"default=info"`
-	DomainName string
+	LogLevel string `envconfig:"default=info"`
+
+	// FreemiumProviders is a comma separated list of providers for freemium
+	FreemiumProviders string `envconfig:"default=aws"`
```
[the remark this answers is not in the document]
Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/cmd/broker/main.go]: the envconfig library allows to define slices which are automatically decoded.

**Review by ksputo — APPROVED** [review]
lgtm

**Pull request merged**